From 6b2619a7442eb29e12ead17d8d9c5a407165a860 Mon Sep 17 00:00:00 2001
From: Florian Sylvestre <fsylvestre@baylibre.com>
Date: Wed, 22 Mar 2023 14:23:46 +0100
Subject: [PATCH 09/24] Revert commit bdc7d6f3d8debbcf66761afec01a596909249db5

tlp_invoque extension definition seems to be useless: so keep default behavior
---
 goil/templates/build/Makefile.goilTemplate | 2 +-
 goil/templates/build/build_py.goilTemplate | 2 +-
 goil/templates/build/make_py.goilTemplate  | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/goil/templates/build/Makefile.goilTemplate b/goil/templates/build/Makefile.goilTemplate
index 13d2ef5..2e6285c 100755
--- a/goil/templates/build/Makefile.goilTemplate
+++ b/goil/templates/build/Makefile.goilTemplate
@@ -62,7 +62,7 @@ end foreach
 if OS::SYSTEM_CALL then
 %
 OIL_GENERATED_FILES += tpl_dispatch_table.c
-OIL_GENERATED_FILES += tpl_invoque.% !EXTENSION %
+OIL_GENERATED_FILES += tpl_invoque.S
 %
 else
 %
diff --git a/goil/templates/build/build_py.goilTemplate b/goil/templates/build/build_py.goilTemplate
index 64df198..f0da871 100755
--- a/goil/templates/build/build_py.goilTemplate
+++ b/goil/templates/build/build_py.goilTemplate
@@ -408,7 +408,7 @@ cSourceList.append(projfile.ProjectFile("% !PROJECT %/tpl_app_config.c"))
 if OS::SYSTEM_CALL then
 %
 cSourceList.append(projfile.ProjectFile("% !PROJECT %/tpl_dispatch_table.c"))
-cSourceList.append(projfile.ProjectFile("% !PROJECT %/tpl_invoque.% !EXTENSION %"))%
+cSourceList.append(projfile.ProjectFile("% !PROJECT %/tpl_invoque.S"))%
 else
 %
 cSourceList.append(projfile.ProjectFile("% !PROJECT %/tpl_os.c"))%
diff --git a/goil/templates/build/make_py.goilTemplate b/goil/templates/build/make_py.goilTemplate
index 3d1b059..f55c842 100755
--- a/goil/templates/build/make_py.goilTemplate
+++ b/goil/templates/build/make_py.goilTemplate
@@ -123,7 +123,7 @@ sourceList.append("% !PROJECT %/tpl_app_config.c")
 if OS::SYSTEM_CALL then
 %
 sourceList.append("% !PROJECT %/tpl_dispatch_table.c")
-sourceList.append("% !PROJECT %/tpl_invoque.% !EXTENSION %")%
+sourceList.append("% !PROJECT %/tpl_invoque.S")%
 else
 %
 sourceList.append("% !PROJECT %/tpl_os.c")%
-- 
2.34.1

